opennms-base-assembly/src/main/filtered/etc/syslog/OpenWrt.syslog.xml

```diff
@@ -0,0 +1,108 @@
+            <match type="substr" expression="shutdown" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/SystemStopping</uei>
+        </ueiMatch>
+        <ueiMatch>
+            <!-- OpenWrt 15.05 version -->
+            <severity>Informational</severity>
+            <process-match expression="^procd$" />
+            <match type="substr" expression="shutdown" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/SystemStopping</uei>
+        </ueiMatch>
+
+        <!-- package ddns-scripts syslog messages -->
+        <ueiMatch>
+            <process-match expression="^ddns-scripts-.*$" />
+            <match type="substr" expression="failed" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/ddns-scripts/DDNSIPUpdateFailed</uei>
+        </ueiMatch>
+        <ueiMatch>
+            <process-match expression="^ddns-scripts-.*$" />
+            <match type="substr" expression="successful" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/ddns-scripts/DDNSIPUpdateSuccessful</uei>
+        </ueiMatch>
+
+        <!-- package mwan3 syslog messages -->
+        <ueiMatch>
+            <severity>Informational</severity>
+            <process-match expression="^mwan3track$" />
+            <match type="regex" expression="^Lost [0-9]+ ping\(s\) on interface (.*)$" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/mwan3/LostPingsOnInterface</uei>
+            <parameter-assignment matching-group="1" parameter-name="interfaceName" />
+        </ueiMatch>
+        <ueiMatch>
+            <severity>Informational</severity>
+            <process-match expression="^mwan3track" />
+            <match type="regex" expression="^Interface (.*) is online$" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/mwan3/OnlineInterface</uei>
+            <parameter-assignment matching-group="1" parameter-name="interfaceName" />
+        </ueiMatch>
+        <ueiMatch>
+            <severity>Informational</severity>
+            <process-match expression="^mwan3track$" />
+            <match type="regex" expression="^Interface (.*) is offline$" />
+            <uei>uei.opennms.org/vendor/openwrt/syslog/mwan3/OfflineInterface</uei>
+            <parameter-assignment matching-group="1" parameter-name="interfaceName" />
+        </ueiMatch>
+
+        <!-- The only syslog sending mechanism OpenWrt has is to send ALL syslog messages to a remote syslog server. This results in a high
+             volume of syslog messages being received and stored in OpenNMS. To avoid this, the statement below will discard (and not save)
+             all syslog messages received from the specified devices below (other than those already captured above).
+               OpenWrt 12.09: list the device by its hostname (lowercase)
+               OpenWrt 14.07 and later: list the device by the source IP address it uses to send the syslog packet -->
+        <ueiMatch>
+            <hostname-match expression="(openwrt01|192\.0\.2\.1).*" />
```

Comment:
@indigo423 Hi Ronny, yes, the hostname-match expression is something the end-user sysadmin would typically modify by adding the hostname/IP address of their OpenWrt device once they have configured OpenWrt to send syslog messages to OpenNMS. The problem is that OpenWrt has no syslog filtering on its send side, so OpenNMS gets all the syslog messages from the whole OS. The hostname-match above is just to avoid the event list clutter that results in OpenNMS by dropping all syslog messages after the specific desired ones have already been handled. -Tim